packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -306,6 +326,14 @@ class ThemeData {
   /// icons placed on top of the primary color (e.g. toolbar text).
   final Brightness primaryColorBrightness;
 
+  /// A lighter version of the primary color, used for showing subtle differences
```

Comment:
Here and below: I think it would be enough to just say "A lighter/darker version of the primary color".

Reply:
OK.